internal/cltest/fixtures.go

```diff
@@ -22,7 +22,7 @@ import (
 
 func NewJob() models.Job {
 	j := models.NewJob()
-	j.Tasks = []models.Task{{Type: "NoOp"}}
+	j.Tasks = []models.Task{NewTask("NoOp", `{}`)}
```

Comment:
Doesn't this do the same thing? I'm hoping the switch to `NewTask` isn't needed but perhaps we want to be more explicit.

Reply:
It doesn't, the old way was failing silently; there would be no type on the task when it was pulled back from the DB. The `type` field needs to be set on `Params`.

Reply:
Can we update our `JSON` to be resilient and have a valid zero value that's serializable to bolt?

Reply:
I believe that's already the case, the second param was just there because the test helper requires it.

Reply:
made the test helper more flexible.